From 31f4a08c068c71b31c36d8c862ebd1a49fefa676 Mon Sep 17 00:00:00 2001
Message-Id: <patch-1.1-31f4a08c068-20230207T161453Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 20 Jan 2023 00:19:12 +0100
Subject: [PATCH] config.h: remove unused git_configset_add_parameters()
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This function was removed in ecec57b3c97 (config: respect includes in
protected config, 2022-10-13), but its prototype was left here.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---

A trivial cleanup for a change new in v2.39.0.

 config.h | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/config.h b/config.h
index ef9eade6414..7606246531a 100644
--- a/config.h
+++ b/config.h
@@ -447,15 +447,6 @@ void git_configset_init(struct config_set *cs);
  */
 int git_configset_add_file(struct config_set *cs, const char *filename);
 
-/**
- * Parses command line options and environment variables, and adds the
- * variable-value pairs to the `config_set`. Returns 0 on success, or -1
- * if there is an error in parsing. The caller decides whether to free
- * the incomplete configset or continue using it when the function
- * returns -1.
- */
-int git_configset_add_parameters(struct config_set *cs);
-
 /**
  * Finds and returns the value list, sorted in order of increasing priority
  * for the configuration variable `key` and config set `cs`. When the
-- 
2.39.1.1430.gb2471c0aaf4

